protonvpn_cli/utils.py

```diff
@@ -167,6 +167,18 @@ def get_fastest_server(server_pool):
     return fastest_server
 
 
+def get_default_nic():
+    """Find and return the default network interface"""
+    default_route = subprocess.run(
+        "ip route show | grep default",
+        stdout=subprocess.PIPE, shell=True
+    )
+
+    # Get the default nic from ip route show output
+    default_nic = default_route.stdout.decode().strip().split()[4]
+    return default_nic
```

Comment:
So this technique I guess it's not the best right? Because there might be more than one interface being enabled? Then we should likely display a disclaimer.

Reply:
I guess, yea. With multiple NICs many things in terms of Kill Switch and leak protection would fail. But afaik that's nothing unique to Linux, the Kill Switch on Windows and Mac also have some edge cases when using it on a system with multiple active nics that might even switch network etc.

Regarding disclaimer, when should that be printed? I don't think it's great to display it when multiple NICs are detected, because how do you detect that? How to determine if it's actually a physical NIC that routes network to the outside or a virtual NIC for stuff like docker, virtualbox or network namespaces.

Reply:
Maybe next to kill switch option when you enable it:
> Please enter your choice or leave empty to quit: 5
> The Kill Switch will block all network traffic
> if the VPN connection drops unexpectedly.

E.g. note that kill-switch correct execution assumes only one network interface is being activated. or something similar.

Reply:
Sounds good, I'll add this. It is something for the other PR, though 🙂

Reply:
Updated here: https://github.com/ProtonVPN/protonvpn-cli-ng/blob/73f572a3a5eff8438ccf9fa4544fa57908ba4a1a/protonvpn_cli/cli.py#L548